src/components/GoogleMaps/index.js

```diff
@@ -20,7 +20,8 @@ export const GoogleMaps = (props) => {
     setNearBusinessList,
     noDistanceValidation,
     businessZones,
-    fillStyle
+    fillStyle,
+    isCallcenterAddressMap
```

Comment:
improve this variable name, I see this var is for blocking alerts and blocking the map interval. If is necessary you can create 2 variables for any block needs.